src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -22,19 +30,55 @@
    * the database and checking to make sure its valid and not null
    * and sending this information to the webpage
    */
+   
+  /** Store class that gives access to Users. */
+  private UserStore userStore;
+  
+  /** Store class that gives access to Conversations. */
+  private ConversationStore conversationStore;
   
   /** This is for the inital setup of the admin page */
   @Override
   public void init() throws ServletException {
     super.init();
+    
+    /** This is for getting the current instance of users */
+    setUserStore(UserStore.getInstance());
+    
+    /** This is for getting the current instance of conversations */
+    setConversationStore(ConversationStore.getInstance());
```

Comment:
A note on comments: I only recommend them when it's not immediately clear to someone reading your code what the purpose of your code is / what is going on. Here, it's pretty obvious. Therefore, I'd say it's actually better to let your code speak for itself here and delete them.

I'd go through some of your other comments as well and evaluate whether you need them. In college they'll make you comment `*everything*`, but, in reality, it's not always the case in industry.